.gitignore

```diff
@@ -17,6 +17,7 @@ local.properties
 .class
 .settings/
 .loadpath
+*.lock
```

Comment:
`yarn.lock` shouldn't be ignored even though we don't use *__yarn__*.

Reply:
Thanks, it has been modified